src/Exception/DeadConnectionException.php

```diff
@@ -0,0 +1,11 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Keboola\SnowflakeTransformation\Exception;
+
+use Keboola\Component\UserException;
+
+class DeadConnectionException extends UserException
```

Comment:
a tady taky tak, + by se to melo upravit v php-component

Reply:
ta se už nepoužívá vůbec, zapomněl jsem jí smazat